lib/oban/job.ex

```diff
@@ -295,8 +295,9 @@ defmodule Oban.Job do
   end
 
   defp to_clean_string(value) do
-    value
-    |> to_string()
-    |> String.trim_leading("Elixir.")
+    case value |> to_string() do
```

Comment:
```suggestion
    case to_string(value) do
```

Reply:
I found this single step pipe in other files as well.

Do you want to drop such single step pipe? Then I'll create another MR for that using [`Credo.Check.Readability.SinglePipe`](https://github.com/rrrene/credo/blob/v1.4.0/lib/credo/check/readability/single_pipe.ex)

Reply:
Nicely spotted. I feel like there was an issue with that in nimble_parsec in the past, but it isn't an ussue now. I've fixed it with 0014bdf